vision/system-test/quickstart.test.js

```diff
@@ -16,46 +16,17 @@
 'use strict';
 
 const path = require(`path`);
-const proxyquire = require(`proxyquire`).noPreserveCache();
-const sinon = require(`sinon`);
 const test = require(`ava`);
 const tools = require(`@google-cloud/nodejs-repo-tools`);
 
-const vision = proxyquire(`@google-cloud/vision`, {})();
-
 test.before(tools.stubConsole);
 test.after.always(tools.restoreConsole);
 
-test.cb(`should detect labels`, (t) => {
-  const filePath = path.join(__dirname, `../resources/wakeupcat.jpg`);
-  const expectedFileName = `./resources/wakeupcat.jpg`;
-  const visionMock = {
-    detectLabels: (_fileName) => {
-      t.is(_fileName, expectedFileName);
-
-      return vision.detectLabels(filePath)
-        .then(([labels]) => {
-          t.true(Array.isArray(labels));
-
-          setTimeout(() => {
-            try {
-              t.is(console.log.callCount, 6);
-              t.deepEqual(console.log.getCall(0).args, [`Labels:`]);
-              labels.forEach((label, i) => {
-                t.deepEqual(console.log.getCall(i + 1).args, [label]);
-              });
-              t.end();
-            } catch (err) {
-              t.end(err);
-            }
-          }, 200);
-
-          return [labels];
-        });
-    }
-  };
+const cmd = `node quickstart.js`;
```

Comment:
We pass it explicitly because that's how all of our "quickstart" samples were designed. For the quickstarts we chose to be explicit about what project is being used.